code/datums/dna.dm

```diff
@@ -272,6 +274,20 @@
 	return
 
 /////////////////////////// DNA MOB-PROCS //////////////////////
+/datum/dna/proc/update_body_size()
+	if(!holder)
+		return
+	var/static/list/feat2num = list("Normal" = BODY_SIZE_NORMAL, "Tall" = BODY_SIZE_TALL, "Short" = BODY_SIZE_SHORT)
```

Comment:
Why do you have both a global list of string options and a static assoc list of string/define?

Reply:
bosnia told me to make it static.

They need to be strings for user customization, and numbers for, you know, math.

Reply:
But like you can't just have one global assoc of string/value? IIRC in BYOND if you create an input window of an assoc list it lists the keys and sets the choice to the value.

Reply:
yes